### PR TITLE
Update logic for reading user data from db

### LINKED ISSUE #329: UI updates for seekWorkPlan
- Update logic in RetroCertsCertificationPage/index.js to account for arrays in seekWorkPlan when determining which questions to display (lines 103-113, 161, 239)

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR updates the logic for reading from the db to account for the following:

Most (99.99%) users have the same seekWorkPlan for all weeks in weeksToCertify, in which case their seekWorkPlan is a one element array. The rest have an array whose length matches the weeksToCertify length.

Note that this logic should continue to work with all existing test data created under the prior assumption that all seekWorkPlan arrays would be the same length as the weeksToCertify array for that row.

===

Resolves #329

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: src/client/pages/RetroCertsCertificationPage/index.js [hunk 1]
```diff
@@ -75,8 +75,14 @@ function RetroCertsCertificationPage(props) {
...
 
-  const weekSeekWorkPlan =
-    userData.seekWorkPlan[userData.weeksToCertify.indexOf(weekIndex)];
+  // Most (99.99%) users have the same seekWorkPlan for all weeks in
+  // weeksToCertify, in which case their seekWorkPlan is a one element array.
+  // The rest have an array whose length matches the weeksToCertify length.
+  const seekWorkPlanIndex =
+    userData.seekWorkPlan.length === 1
+      ? 0
+      : userData.weeksToCertify.indexOf(weekIndex);
```
Comment (idealisms) [inline src/client/pages/RetroCertsCertificationPage/index.js:84]: Oh, I think you could also do weekForUser - 1 for seekWorkPlanIndex, but the old code works as well.

Comment (kalvinwang) [reply src/client/pages/RetroCertsCertificationPage/index.js:84]: Do you mean: `userData.seekWorkPlan.length === 1 ? 0 : weekForUser - 1`?

Comment (kalvinwang) [reply src/client/pages/RetroCertsCertificationPage/index.js:84]: Yes you do, good idea thanks!

**Comment by kalvinwang** [comment]
@sharonwarner there's no new field that I added! if you mean renaming seekWorkPlan to programPlan, I haven't done that yet, will do in a separate PR if I have time

File: src/client/pages/RetroCertsCertificationPage/index.js (hunk 1 above)
Comment (idealisms) [reply src/client/pages/RetroCertsCertificationPage/index.js]: Yes, sorry, didn't see this.

**Pull request merged**